index.html

```diff
@@ -194,20 +209,20 @@ <h3>
 
   <div class="mana">
     <div class="mana0">
-    <h3>Languages  <i class="fas fa-angle-right" id="chev"></i> </h3>
+    <h3 class="lang-hd">Languages  <i class="fas fa-angle-right" id="chev"></i> </h3>
     </div>
     <div class="lang-list">
     <ul class="pang">
-      <li class="pang-item"> <img src="./img/Ellipse 1.png" alt="" class="im"> &nbsp; &nbsp; JavaScript</li>
-      <li class="pang-item"> <img src="./img/Ellipse 1 (1).png" alt="" class="im"> &nbsp; &nbsp;  HTML</li>
-      <li class="pang-item"> <img src="./img/Ellipse 1 (2).png" alt="" class="im"> &nbsp; &nbsp; CSS</li>
+      <li class="pang-item"> <img src="./img/Ellipse 1.png" alt="" class="im"> <p class="lang-p">JavaScript</p></li>
+      <li class="pang-item"> <img src="./img/Ellipse 1 (1).png" alt="" class="im"> <p class="lang-p">HTML</p>  </li>
+      <li class="pang-item"> <img src="./img/Ellipse 1 (2).png" alt="" class="im">  <p class="lang-p">CSS</p></li>
     </ul>
   </div>
 
   <div class="nrop">
-    <p>Frameworks <i class="fas fa-angle-right"></i> </p>
+    <p class="lang-hd">Frameworks <i class="fas fa-angle-right"></i> </p>
     <hr>
-    <p>Skills &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <i class="fas fa-angle-right"></i></p>
+    <p class="lang-hd">Skills &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <i class="fas fa-angle-right"></i></p>
     <hr>
   </div>
 </section>
```

Comment:
Great work sticking to the design requirements so far. 🎨 I however have some few change suggestions to make the design complete. ♻️ 

Your webpage seem not to be completely responsive and has different margins in various screen sizes. In a desktop view, the `About` and `Contact` sections seem to be align to the left side of the page. 

![about](https://github.com/Danny-imo/my-project/assets/117144028/289ee8c1-5f2f-47f0-b796-10aa679b515c)

- [ ] Please make sure that your sections are centered in all screen sizes as it ensures consistency in design, which is the core factor in responsive design. 🎯